### DISCUSSION TITLE
In-between labels for Timeline items

### DISCUSSION THREAD

**Discussion by keraf** [mantinedev/mantine#6525] [Q&A]
I love the Timeline component, but I feel it's missing one really useful feature: labels in-between items.

These can be used to describe what is happening between two points. It would be even nicer if these can appear on the opposite side of the items. That would allow to create UIs like SkyScanner's flight details for example where both items are the departure and arrival, and the label the travel time:
![image](https://github.com/user-attachments/assets/bf78fe1e-f073-4551-a882-167b9a060438)

I'd be happy to give it a go and submit a PR if it's a fit for the project and can be useful for others too.

**Comment by rtivital** [comment]
You are welcome to submit a PR with this feature
